src/SFA.DAS.AdminService.Web.Tests/Controllers/Gateway/RoatpOrganisationCriminalComplianceChecksControllerTests.cs

```diff
@@ -54,28 +54,41 @@ public void Setup()
             _controller = new RoatpOrganisationCriminalComplianceChecksController(_applyApiClient.Object, _contextAccessor.Object, _gatewayValidator.Object, _orchestrator.Object, _logger.Object);
         }
 
-        [Test]
-        public void Composition_with_creditors_check_returns_view()
+        [TestCase(GatewayPageIds.CriminalComplianceOrganisationChecks.CompositionCreditors)]
+        [TestCase(GatewayPageIds.CriminalComplianceOrganisationChecks.FailedToRepayFunds)]
+        [TestCase(GatewayPageIds.CriminalComplianceOrganisationChecks.ContractTermination)]
+        [TestCase(GatewayPageIds.CriminalComplianceOrganisationChecks.ContractWithdrawnEarly)]
+        [TestCase(GatewayPageIds.CriminalComplianceOrganisationChecks.RemovedRoTO)]
+        [TestCase(GatewayPageIds.CriminalComplianceOrganisationChecks.FundingRemoved)]
+        [TestCase(GatewayPageIds.CriminalComplianceOrganisationChecks.RemovedRegister)]
+        [TestCase(GatewayPageIds.CriminalComplianceOrganisationChecks.IttAccreditation)]
+        public void Criminal_compliance_check_returns_view(string gatewayPageId)
```

Comment:
Very minor but I'd rather the test description reflected the behaviour rather than the code.  In this case it'd be as simple as saying that Criminal_compliance_check_is_displayed and the next test Criminal_compliance_check_assessment_is_saved